CHANGELOG.md

```diff
@@ -21,6 +21,9 @@
   PR [#2561](https://github.com/realm/realm-core/pull/2561).
 * New replication instruction: instr_AddRowWithKey
 * Add the old table size to the instr_TableClear replication instruction.
+* Throw a MaxAddressBreached exception in some places instead of wrapping around.
```

Comment:
I tried to clarify the changelog (see updates) let me know if you have other suggestions there. It could be debatable, but I think bindings should pass the error along to apps to clear up unneeded space.